api/src/middlewares/services/SubmitService/SubmitService.ts

```diff
@@ -0,0 +1,47 @@
+import logger, { Logger } from "pino";
+import { FileService } from "../file";
+import { SESService } from "../EmailService/SESService";
+import { FormDataBody } from "../../../types";
+import { flattenQuestions } from "../../../handlers/forms/helpers/flattenQuestions";
+import { isFieldType } from "../../../utils";
+import { isNotFieldType } from "../../../utils";
+const { customAlphabet } = require("nanoid");
+
+const nanoid = customAlphabet("1234567890ABCDEFGHIJKLMNPQRSTUVWXYZ-_", 10);
+export class SubmitService {
+  logger: Logger;
+  fileService: FileService;
+  emailService: SESService;
+
+  constructor({ fileService, emailService }) {
+    this.logger = logger().child({ service: "Submit" });
+    this.fileService = fileService;
+    this.emailService = emailService;
+  }
+
+  generateId() {
+    return nanoid();
+  }
+
+  /**
+   * @throws ApplicationError
+   */
+  async submitForm(formData: FormDataBody) {
```

Comment:
🧅s all the way down. 

trying to keep the route handler calls as simple as possible, they should be interfacing with this.

route handler will try to coordinate between the different services. 

the other services are available however, but `submitService.submitForm(allTheDataIveGot)` makes it simpler.